test/extensions/filters/http/ext_authz/ext_authz_fuzz_test.cc

```diff
@@ -92,7 +89,7 @@ DEFINE_PROTO_FUZZER(const envoy::extensions::filters::http::ext_authz::ExtAuthzT
     return;
   }
 
-  static FuzzerMocks mocks;
+  FuzzerMocks mocks;
```

Comment:
Thanks for working on this!
A few high nit comments first: Is this change intentional?

Reply:
Yes, but I realized the real problem is just the stateful mocks being static. I've refactored a bit. Hopefully my intent is more clear.